src/image_export_file.py

```diff
@@ -88,6 +98,14 @@ def file_extract(
             # Execute the command and block until it finishes.
             subprocess.call(command)
 
+            if compress_output:
+                # Compress the extracted files into zip archive and remove the extracted files.
+                with tempfile.TemporaryDirectory() as temp_dir:
```

Comment:
This can be slow if the temporary directory is created on a different filesystem than the extracted files, for example when we store files in NFS but the tmp directory is local to the worker. Create a temporary directory along side the export directory instead and then remove it manually when done.